src/realm/obj.cpp

```diff
@@ -1774,7 +1774,7 @@ inline void nullify_set(Obj& obj, ColKey origin_col_key, T target)
 
 void Obj::nullify_link(ColKey origin_col_key, ObjLink target_link)
 {
-    ensure_writeable();
+    // ensure_writeable();
```

Comment:
@jedelbo shouldn't this be deleted or an explanation added?

Reply:
This can be deleted. Just forgot.